app/views/hosts/show.html.erb

```diff
@@ -65,7 +65,7 @@
           <%= spinner(_('Loading VM information ...')) %>
         </div>
       <% end %>
-      <div id="nics" class="tab-pane" data-ajax-url='<%= nics_host_path(@host)%>' data-on-complete='tfm.tools.activateTooltips'>
+      <div id="nics" class="tab-pane" data-ajax-url='<%= nics_host_path(@host)%>'>
```

Comment:
This tab's tooltips are already initialized when the page loads in the new version.